src/GitLink/Extensions/PdbExtensions.cs

```diff
@@ -68,8 +68,15 @@ public static IEnumerable<string> FindMissingOrChangedSourceFiles(this PdbFile p
 
         private static bool AreEqualBuffers(byte[] first, byte[] second)
         {
-            Argument.IsNotNull(nameof(first), first);
-            Argument.IsNotNull(nameof(second), second);
+            if (first == null && second == null)
```

Comment:
I mean the result of the null check on first (because we do it twice), but again that's probably premature optimization as well.